servicetalk-opentelemetry-http/build.gradle

```diff
@@ -47,6 +46,7 @@ dependencies {
   testImplementation "com.fasterxml.jackson.core:jackson-databind"
   testImplementation "io.opentelemetry:opentelemetry-sdk-trace"
   testImplementation "io.opentelemetry:opentelemetry-sdk-testing"
+  testImplementation 'io.opentelemetry.semconv:opentelemetry-semconv:1.23.1-alpha'
```

Comment:
Why this version needs to be different from what is defined by the `opentelemetry-instrumentation-bom-alpha`?